src/Http/composer.json

```diff
@@ -15,12 +15,10 @@
     ],
     "require": {
         "php": "^8.0",
-        "php-http/client-implementation": "^1.0",
-        "php-http/discovery": "^1.6",
-        "php-http/httplug": "^1.0 || ^2.0",
-        "php-http/message-factory": "^1.0.2",
-        "psr/http-message": "^1.0 || ^2.0",
-        "psr/http-message-implementation": "^1.0",
+        "php": "^7.4 || ^8.0",
```

Comment:
This looks like an issue while rebasing. Please remove this line and support php8 only.